src/plugins/controls/public/react_controls/control_group/utils/serialization_utils.ts

```diff
@@ -43,7 +43,7 @@ export const deserializeControlGroup = (
     autoApplySelections:
       typeof state.rawState.showApplySelections === 'boolean'
         ? !state.rawState.showApplySelections
-        : false, // Rename "showApplySelections" to "autoApplySelections"
+        : true, // Rename "showApplySelections" to "autoApplySelections"
```

Comment:
The default should have always been `true` - i.e. by default, we **do** want to auto apply selections.